sklearn/metrics/classification.py

```diff
@@ -1348,6 +1348,66 @@ def recall_score(y_true, y_pred, labels=None, pos_label=1, average='binary',
     return r
 
 
+def balanced_accuracy_score(y_true, y_pred, sample_weight=None):
+    """Compute the balanced accuracy
+
+    The balanced accuracy is used in binary classification problems to deal
+    with imbalanced datasets. It is defined as the arithmetic mean of sensitivity
+    (true positive rate) and specificity (true negative rate), or the average
+    accuracy obtained on either class.
```

Comment:
Either use "recall" here or be explicit that it's "on either class's gold standard instances" or something.